### PR TITLE
Initial work for social media for company profiles

### PR DESCRIPTION
**Pull request by amillion3**
Added facebook, twitter, instagram and a website field for companies.

### CODE REVIEW

File: client/components/Companies/Form/index.js [hunk 1]
```diff
@@ -71,6 +71,27 @@ export const CompanyForm = () => {
           label="Description"
           help={descHelp()}
         />
+        <Field
+          name="facebook"
+          type="text"
+          component={Input}
+          label="Facebook Page"
+          help={descHelp()}
```
Comment (egdelwonk) [inline client/components/Companies/Form/index.js]: You'll want to make each of these fields have their own help element (maybe?)

File: server/controllers/company.js [hunk 2]
```diff
@@ -6,6 +6,9 @@ exports.create = async function(req, res, next) {
   req.assert("phone", "Please enter a phone number").notEmpty();
   req.assert("size", "Please enter a company size").isInt();
   req.assert("description", "Please enter a description").notEmpty();
+  req.assert("facebook", "Please enter a Facebook Page").notEmpty();
+  req.assert("twitter", "Please enter a Twitter Page").notEmpty();
+  req.assert("website", "Please enter a Website").notEmpty();
```
Comment (egdelwonk) [inline server/controllers/company.js]: You can add `.isURL()` to ensure website is a valid url.

**Comment by egdelwonk** [comment]
Good first pass. I think your next steps will be:

1. To create the database tables and a migration. 
2. Display these new values on the Company profile

**Comment by egdelwonk** [comment]
Hey @amillion3 did you make any more progress on this?

**Comment by amillion3** [comment]
I have not made any progress. I'm not sure how to create columns, migrate the db, etc. I'd be happy to learn, but I haven't done anything like that before.

**Comment by egdelwonk** [comment]
Hey Andy, going to close this for now - let me know if you end up making any progress.

Cheers,

Will

**Pull request closed without merging**